02-documentation/developer-guides/building-a-nextjs-blog-with-squidex.md

```diff
@@ -0,0 +1,588 @@
+
+Check the _Allow anonymous access_ checkbox to allow this client to be query published blog posts without an acess token.
+
+Now you have created a new client and allowed anonymous access to published blog posts, you will create a file containing environment variables for your Next.js app. Create a `.env.local` file in the `squidex-blog` folder and add the following contents
+
+```
+SQUIDEX_API_URL=https://cloud.squidex.io/api/content/squidex-blog/graphql
+```
+
+Replace `squidex-blog` in the value of SQUIDEX_API_URL with the name of your app on Squidex.
+
+Restart the development server to load the environment variables from `.env.local`. Press the combination `Ctrl/CMD + C` on your terminal to stop the development server, then restart it by running:
+
+```bash
+yarn dev
+```
+
+You will see a message on the terminal showing that environment variables have been loaded from `.env.local`.
+
+Now you have set up the environment variables for your Next.js blog, you will create a helper function that you will use to make GraphQL queries to Squidex.
+
+Create a `lib` folder in `squidex-blog` and in this `lib` folder, create a `squidex.js` file. Add the following contents to the `squidex-blog/lib/squidex.js` file:
+
+```js
+export default async function fetchAPI(query, { variables } = {}) {
+  const res = await fetch(process.env.SQUIDEX_API_URL, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      query,
+      variables,
+    }),
+  });
+  const json = await res.json();
+
+  if (json.errors) {
+    console.error(json.errors);
+    throw new Error("Failed to fetch API");
+  }
+
+  return json.data;
+}
+```
+
+This exports a `fetchAPI` function that receives a GraphQL query and its variables as parameters, queries the Squidex API, and returns the result.
+
+You are now ready to fetch content from Squidex and display it in your Next.js blog.
+
+## Step 6: Displaying blog posts on home page
+
+In this step, you will fetch the titles of and links to your blog post and display them on the homepage of your blog.
+
+Replace the contents of `squidex-blog/pages/index.js` with this:
+
+```jsx
+import Head from "next/head";
+
+export default function Home() {
+  return (
+    <div>
+      <Head>
+        <title>My blog</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+    </div>
+  );
+}
+```
+
+Open `http://localhost:3000` in your browser if you haven't already to continue previewing changes.
```

Comment:
[...] in your browser, if you haven't already, to continue previewing changes [...] <-- commas missing